airflow/providers/google/cloud/operators/gcs.py

```diff
@@ -16,12 +16,15 @@
 # specific language governing permissions and limitations
 # under the License.
 """This module contains a Google Cloud Storage Bucket operator."""
+import datetime
 import subprocess
 import sys
 import warnings
-from tempfile import NamedTemporaryFile
+from pathlib import Path
+from tempfile import NamedTemporaryFile, TemporaryDirectory
 from typing import Dict, Iterable, List, Optional, Sequence, Union
 
+import dateutil.tz
```

Comment:
```suggestion
from airflow.settings import TIMEZONE
```
I think using this would be better. What do you think?

Reply:
Good question - when we compare against GCS modified time we want to use UTC (I think GCS reports in UTC). But maybe for the transfer function it would make more sense to get the timestamps in the local time. What do you think?

Reply:
I guess `airflow.timezone.utc`  would be better for the GCS modified time then, just for uniformity.

Reply:
Good point. I took this from an existing hook - but the next commit has it fixed in both spots.